dashboard/app/controllers/application_controller.rb

```diff
@@ -177,9 +177,10 @@ def milestone_response(options)
       response[:message] = 'no script provided'
     end
 
+    response[:phone_share_url] = send_to_phone_url
```

Comment:
`send_to_phone_url` is a rails shortcut which evaluates to a url ("/sms/send") pointing to `def send_to_phone` below.